internal/rbac/main.go

```diff
@@ -143,8 +152,11 @@ func Roles(user types.User) ([]RoleName, error) {
 
 // Does the user have a role?
 func HasRole(user types.User, role RoleName) (bool, error) {
-	hasRole, err := enforcer.HasRoleForUser(user.ID.String(), string(role))
```

Comment:
`HasRoleForUser` doesn't include implicit ones, so switch to `slices.Contains` on the implicit ones (i.e. include inherited)